src/openrct2/ReplayManager.cpp

```diff
@@ -77,6 +79,14 @@ namespace OpenRCT2
         }
     };
 
+    struct ReplayRecordFile
+    {
+        uint32_t magic;
+        uint16_t version;
+        uint64_t uncompressedSize;
```

Comment:
`uint32_t`?

Reply:
Sure.

Reply:
Actually nevermind, would result in a useless cast from zlib.